LEAF_Nexus/sources/Employee.php

```diff
@@ -887,7 +887,7 @@ public function getPositions(int|string $empUID): int|array|bool
      * @param string $login
      * @param bool $searchDeleted
      */
-    public function lookupLogin($login, bool $searchDeleted = false): array
+    public function lookupLogin($login, bool $searchDeleted = true): array
```

Comment:
This is going to need some good regression testing.  Changing a default to the opposite can have a slew of unintended consequences.

Reply:
This should not be changed. The endpoint call on the account updater page should be updated to pull deleted users. That will ensure the rest of the site works as it did before.